components/renku_data_services/migrations/versions/08ac2714e8e2_add_template_flag_to_projects.py

```diff
@@ -0,0 +1,28 @@
+"""add template flag to projects
+
+Revision ID: 08ac2714e8e2
+Revises: ea52d750e389
+Create Date: 2024-11-19 13:17:22.222365
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "08ac2714e8e2"
+down_revision = "ea52d750e389"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("projects", sa.Column("template", sa.Boolean(), default=False, nullable=False), schema="projects")
```

Comment:
Suggestions for a better name are welcome! We can name it `is_template` which shows the intention better.

Reply:
`is_template` does sound better to me